src/Styleguide/Components/__tests__/ReadMore.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { mount } from "enzyme"
+import React from "react"
+import { ReadMore } from "../ReadMore2"
+
+describe("ReadMore", () => {
+  const copy =
+    "Donald Judd regarded as one of the most significant American artists of the post-war period, is perhaps best-known for the large-scale outdoor installations and long, spacious interiors he designed in Marfa. Donald Judd, widely regarded as one of the most significant American artists of the post-war period, is perhaps best-known for the large-scale outdoor installations and long, spacious interiors he designed in Marfa."
+
+  it("it truncates text", () => {
+    const wrapper = cap => mount(<ReadMore maxChars={cap} text={copy} />)
+    expect(wrapper(20).html()).toContain(">Donald Judd regarded<")
```

Comment:
nice use of opening/closing tags to make sure you were testing this properly, vs a 'should include' style matcher, which would have been true even if it wasn't being truncated.